recipes/jupyterlab-plotly-extension/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jupyterlab-plotly-extension" %}
+{% set version = "0.18.2" %}
+{% set sha256 = "f838a57fb3616bc496b2054475cfa627c52d4c5976849f5413d3f0d8ac77d36a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jupyterlab/jupyter-renderers/archive/@jupyterlab/plotly-extension@{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - nodejs
+  host:
+    - jupyterlab
+    - python >=3.5
+  run:
+    - jupyterlab
+    - nodejs
+    - python >=3.5
+
+test:
+  commands:
+    - jupyter labextension list
+    - jupyter lab build
+
+about:
+  home: https://github.com/jupyterlab/jupyter-renderers/blob/master/packages/plotly-extension
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: A JupyterLab extension for rendering Plotly charts
+  dev_url: https://github.com/jupyterlab/jupyter-renderers
+
+extra:
+  recipe-maintainers:
+    - dbast
+    - blink1073
+    - uday1889
```

Comment:
@uday1889 - are you OK with being a maintainer here?

Reply:
@scopatz: Sure. That's fine with me.